HW16/Team.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+#include <vector>
+#include <string>
+#include "Player.h"
+
+class Player;
+class Team {
+private:
+    std::string teamName;
+    int numberOfMembers;
```

Comment:
this value is not needed at all. You can just use players.size() whenever you need the size.